packages/tracing-internal/test/browser/metrics/index.test.ts

```diff
@@ -1,6 +1,24 @@
 import { Transaction } from '../../../src';
 import type { ResourceEntry } from '../../../src/browser/metrics';
 import { _addMeasureSpans, _addResourceSpans } from '../../../src/browser/metrics';
+import { WINDOW } from '../../../src/browser/types';
+
+const mockWindowLocation = {
+  ancestorOrigins: {},
+  href: "https://example.com/path/to/something",
+  origin: "https://example.com",
+  protocol: "https",
+  host: "example.com",
+  hostname: "example.com",
+  port: "",
+  pathname: "/path/to/something",
+  search: "",
+  hash: ""
+} as Window['location'];
+
+WINDOW.location = mockWindowLocation;
```

Comment:
this needs to be put in a `beforeAll` and cleaned up in an `afterAll`

Reply:
done!